extension/content.js

```diff
@@ -307,6 +307,43 @@ $(document).on('keydown', event => {
 	}
 });
 
+function addReactionParticipants() {
+	$('.comment-reactions').each((index, reactionsContainer) => {
+		const $reactionsContainer = $(reactionsContainer);
+		if (!$reactionsContainer.hasClass('has-reactions')) {
+			return;
+		}
+
+		const $reactionButtons = $reactionsContainer.find('.comment-reactions-options .reaction-summary-item[aria-label]');
+
+		$reactionButtons.each((index, element) => {
+			const participants = [];
```

Comment:
Instead of the empty array here, why not just set `const participants` to the return value of the `map` (lines 322-328)